packages/components/psammead-media-player/src/Guidance/index.jsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import { string, node } from 'prop-types';
+import styled from 'styled-components';
+import { C_WHITE } from '@bbc/psammead-styles/colours';
+import { GEL_GROUP_2_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';
+import { mediaIcons } from '@bbc/psammead-assets/svgs';
+import {
+  GEL_MARGIN_BELOW_400PX,
+  GEL_SPACING,
+  GEL_SPACING_HLF,
+  GEL_SPACING_DBL,
+  GEL_SPACING_TRPL,
+} from '@bbc/gel-foundations/spacings';
+import { GEL_LONG_PRIMER } from '@bbc/gel-foundations/typography';
+import { getSansRegular } from '@bbc/psammead-styles/font-styles';
+
+const StyledGuidance = styled.div`
+  ${({ service }) => getSansRegular(service)}
+  ${GEL_LONG_PRIMER};
+
+  strong {
+    font-weight: normal;
+  }
+`;
+
+const GuidanceWrapper = styled.div`
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  background-color: rgba(34, 34, 34, 0.75);
```

Comment:
@jamesdonoh I don't think it's a standard predefined color in `psammead-styles` because of the extra opaccity,  saw a similar usage [in the copyright coomponent](https://github.com/bbc/psammead/blob/latest/packages/components/psammead-copyright/src/index.jsx#L11)